specification.md

```diff
@@ -0,0 +1,151 @@
+# Proveth proof-blob format specification
+
+#### Table of Contents
+
+* [Proof kind](#proof-kind)
+* [Proof of transaction inclusion/exclusion](#proof-of-transaction-inclusionexclusion)
+  * [Examples](#examples)
+    * [Transaction 0 of block 5000000 (mainnet)](#transaction-0-of-block-5000000-mainnet)
+    * [Transaction 109 of block 5000000 (mainnet)](#transaction-109-of-block-5000000-mainnet)
+    * [Transaction 6000 of block 6027762 (mainnet)](#transaction-6000-of-block-6027762-mainnet)
+
+----
+
+## Proof kind
+
+Every proveth proof-blob is an [RLP-encoded](https://github.com/ethereum/wiki/wiki/RLP) list.
+The initial element of the list is a uint that specifies the *kind* of proof the proof-blob represents:
+```
+[kind, actual proof...]
+```
+
+## Proof of transaction inclusion/exclusion
+
+A proof of inclusion proves a statement of the form "the given transaction is present at the given index in the given block". A proof of exclusion proves a statement of the form "there is no transaction at the given index in the given block".
+
+A proof consists of 6 elements:
+- 0: *kind* always equals `1` for this kind of proof
+- 1: *consensus block header*, consisting of
+   ```
+   [<hash of previous block>,
+    <uncles hash>,
+    <miner address>,
+    <state root hash>,
+    <transactions root hash>,
+    <receipts root hash>,
+    <logs bloom filter>,
+    <difficulty>,
+    <number>,
+    <gas limit>,
+    <gas used>,
+    <timestamp>,
+    <extra data>,
+    <mix hash>,
+    <nonce>]
+   ```
+- 2: *transaction index* (zero-based)
```

Comment:
"zero-indexed" not "zero-based"

Reply:
https://en.wikipedia.org/wiki/Zero-based_numbering ?